orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -188,6 +188,15 @@ class ORTTrainerOptions(object):
                             'type': 'integer',
                             'min': 0,
                             'default': 0
+                        },
+                        'propagate_cast_ops_level': {
```

Comment:
FYI, orttrainer API is no longer in use... do we have to update these to make the CI pass?

Reply:
but since you have already done it, let's just keep it here.

Reply:
yes. CI will fail if the defaults values don't match.